include/functions.h

```diff
@@ -836,20 +842,21 @@ void func_800B8C50(Actor* actor, GlobalContext* globalCtx);
 void func_800B8D98(GlobalContext* globalCtx, Actor* actor, f32 param_3, s16 param_4, f32 param_5);
 // void func_800B8DD4(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6);
 // void func_800B8E1C(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
-// void func_800B8E58(void);
-void Audio_PlayActorSound2(Actor* actor, s16 sfxId);
+void func_800B8E58(Actor* actor, u16 sfxId);
```

Comment:
fyi, this function snuck into the recent bg_iknin_susceil PR and may cause a warning. Could you replace it with the following:
`func_800B8E58(player, 0x83E);` -> `func_800B8E58(&player->base, NA_SE_PL_BODY_HIT);`
Sorry I didn't catch it before